File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusColumnVO.kt
package com.sdkwork.backend

data class PlusColumnVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: Int? = null,
    val name: String? = null,
    val description: String? = null,
    val dataType: String? = null,
    val columnType: String? = null,
    val ordinalPosition: Int? = null,
    val isNullable: Boolean? = null,
    val isPrimaryKey: Boolean? = null,
    val isAutoIncrement: Boolean? = null,
    val defaultValue: String? = null,
    val comment: String? = null,
    val characterSet: String? = null,
    val collationRule: String? = null,
    val tableId: Int? = null
)

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_column_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusColumnVO:
    """Column view object"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    name: str = None
    description: str = None
    data_type: str = None
    column_type: str = None
    ordinal_position: int = None
    is_nullable: bool = None
    is_primary_key: bool = None
    is_auto_increment: bool = None
    default_value: str = None
    comment: str = None
    character_set: str = None
    collation_rule: str = None
    table_id: int = None
